### PR TITLE
Update Sprint #19.adoc

### PR DESCRIPTION
**Pull request by rouazana**

### CODE REVIEW

File: Sprint #19.adoc [hunk 1]
```diff
@@ -10,6 +10,7 @@
 ** Update spec with some warnings
 * TASK: Support inline attachment
 * TASK: Support messages with mixed (html and text) content (special multipart case?)
+* TASK: Support attachments naming
```
Comment (mbaechler) [inline Sprint #19.adoc]: We excluded it for sprint 18, do you think it should be included in sprint 19 ?

File: Sprint #19.adoc [hunk 2]
```diff
@@ -23,6 +24,7 @@
 * TASK: handle a max size for upload
 * TASK: support inline attachment
 * TASK: generate mixed content (text + html)?
+* TASK: support attachments naming
```
Comment (mbaechler) [inline Sprint #19.adoc]: What about an attachment uploaded several times with different names ?
I really think the attachment name belongs the message, not the attachment

File: Sprint #19.adoc (hunk 1 above)
Comment (rouazana) [inline Sprint #19.adoc]: Yep, I think it would be good to implement it.

File: Sprint #19.adoc (hunk 2 above)
Comment (rouazana) [inline Sprint #19.adoc]: In fact it has no sense for upload, but it should be done in setMessages.

Comment (mbaechler) [inline Sprint #19.adoc]: please update your PR to make it clear

Comment (rouazana) [inline Sprint #19.adoc]: done

**Pull request merged**